scripts/start_backer.sh

```diff
@@ -9,24 +13,28 @@ else
 fi
 
 mkdir -p $CONFIG_DIR/keri/cf
-echo '{
-    "backer": {
-      "dt": "'$(date -u +"%Y-%m-%dT%H:%M:%S.000000+00:00")'",
-      "curls": ["'$URL':'$PORT'/"]
-    },
-    "dt": "'$(date -u +"%Y-%m-%dT%H:%M:%S.000000+00:00")'",
-    "iurls": [
-    ]
-  }
-  ' > $CONFIG_DIR/keri/cf/backer.json
-echo '{
-    "transferable": false,
-    "wits": [],
-    "icount": 1,
-    "ncount": 1,
-    "isith": "1",
-    "nsith": "1"
-  }' > $CONFIG_DIR/backer_cfg.json
+cat > $CONFIG_DIR/keri/cf/backer.json <<EOF
+{
+  "backer": {
+    "dt": "$(date -u +"%Y-%m-%dT%H:%M:%S.000000+00:00")",
+    "curls": ["tcp://${INTERNAL_HOST}:$((PORT-1))/", "${URL}:${PORT}"]
```

Comment:
@rodolfomiranda 2 points here;

Firstly, I only saw `5666` being defined, so presuming `5665` is just port minus 1 - not sure if that's right or it is hard-coded somewhere.

Secondly - in Docker I use `["tcp://cardano-backer:5665/", "http://localhost:5666"]` and this works if KERIA is running outside of Docker as it accesses via localhost. But for local dev I want to deploy KERIA and the backer both in Docker and point KERIA to the backer via the Docker network - in that case, what should this be for that?

`["tcp://cardano-backer:5665/", "http://localhost:5666"]` doesn't work
`["tcp://cardano-backer:5665/", "http://cardano-backer:5666"]` doesn't work

Also tried with `host.docker.internal`

Reply:
The tcp port is used for direct mode, so it's not needed unless you also start the backer with the `-T port` argument

The URL used there should be accesible by KERIA, if it's inside a docket, that container should be able to resolve it.